distgen/README.md

```diff
@@ -43,3 +43,14 @@ global variables to generate. (See an example of a JSON file at
 package dist
 //go:generate go run github.com/chef/go-libs/distgen global.go dist https://example.com/path/to/glob_dist.json
 ```
+
+### Using an Environment Variable
+
+The usage of an environmental variable allows for better flexibility when dealing with CI systems.
+
+The usage inside of your dist files will not change, but you can pass ENV as of your `go generate` command.
```

Comment:
Thanks for including documentation in your update! Just one suggestion to change as something seemed to be missing.
```suggestion
The usage inside of your dist files will not change, but you can pass ENV as a part of your `go generate` command.
```